Podcast/GoogleLoginViewController.swift

```diff
@@ -0,0 +1,128 @@
+
+import UIKit
+import SwiftyJSON
+import NVActivityIndicatorView
+
+class GoogleLoginViewController: UIViewController, GIDSignInUIDelegate, GIDSignInDelegate {
+
+    var loginButton: GIDSignInButton!
+    var loadingActivityIndicator: NVActivityIndicatorView!
+    var loginBackgroundGradientView: LoginBackgroundGradientView!
+    var podcastLogoView: LoginPodcastLogoView!
+    
+    
+    //Constants
+    var loginButtonViewY: CGFloat = 362
+    var podcastLogoViewY: CGFloat = 140
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    
+        GIDSignIn.sharedInstance().clientID = "724742275706-h8qs46h90squts3dco76p0q6lja2c7nh.apps.googleusercontent.com"
+        GIDSignIn.sharedInstance().delegate = self
+        GIDSignIn.sharedInstance().uiDelegate = self
+        
+        let profileScope = "https://www.googleapis.com/auth/userinfo.profile"
+        let emailScope = "https://www.googleapis.com/auth/userinfo.email"
+
+        GIDSignIn.sharedInstance().scopes.append(contentsOf: [profileScope, emailScope])
+        
+        loginBackgroundGradientView = LoginBackgroundGradientView(frame: view.frame)
+        view.addSubview(loginBackgroundGradientView)
+        
+        loginButton = GIDSignInButton()
+        loginButton.style = .wide
+        loginButton.frame.origin.y = loginButtonViewY
+        loginButton.center.x = view.center.x
+        loginButton.addTarget(self, action: #selector(loginButtonPressed), for: .touchUpInside)
+        view.addSubview(loginButton)
+        
+        podcastLogoView = LoginPodcastLogoView(frame: CGRect(x: 0, y: podcastLogoViewY, width: view.frame.width, height: view.frame.height / 4))
+        podcastLogoView.center.x = view.center.x
+        view.addSubview(podcastLogoView)
+        
+        loadingActivityIndicator = LoadingAnimatorUtilities.createLoadingAnimator()
+        loadingActivityIndicator.center = view.center
+        loadingActivityIndicator.color = .offWhite
+        loadingActivityIndicator.startAnimating()
+        view.addSubview(loadingActivityIndicator)
+        
+        loginButton.isHidden = true
+        
+        signInSilently()
+    }
+    
+    @objc func loginButtonPressed() {
+        loginButton.isHidden = true
+        loadingActivityIndicator.startAnimating()
+    }
+    
+    func signInSilently() {
+        GIDSignIn.sharedInstance().signInSilently()
+    }
+    
+    func sign(_ signIn: GIDSignIn!, didSignInFor user: GIDGoogleUser!, withError error: Error!) {
+        
+        guard error == nil else {
```

Comment:
wait this is incorrect from your merge. We no longer have a google sign in view controller .. its now just `LoginViewController`. You should place the functionality in the same spot there and also include it for failure on facebook login (the spot to add it is very obvious!). Delete this file and add the changes to `LoginViewController`

Reply:
Yeah this file got deleted. I thought I git removed it I specifically didn't add to my commit but I must've accidentally